src/about.js

```diff
@@ -0,0 +1,67 @@
+function setAboutPage() {
+
+  const aboutDiv = createElementWrapper("div", ["card", "mb-3"])
+  const topRowDiv = createElementWrapper("div", ["row", "g-0"])
+  const topCardBodyDiv = createElementWrapper("div", ["card-body"])
+  const h4 = createElementWrapper("h4", ["card-title"])
+  h4.innerHTML = "About"
```

Comment:
anyway every element here could be seperated to function

getHeader

getP1 

and so on..